Onboarding/exploring-tour.md

```diff
@@ -0,0 +1,431 @@
+      headers: [
+        ["Authorization", "Token @config.items.contentrepo_token"]
+      ]
+    )
+
+  image("@image_data.body.meta.download_url")
+
+  buttons(FetchGuidedTourMenu: "@button_labels[0]") do
+    text("@message.message")
+  end
+end
+
+```
+
+# Guided Tour Menu
+
+```stack
+card FetchGuidedTourMenu, then: DisplayGuidedTourMenu do
+  write_result("guided_tour_menu", "yes")
+
+  search =
+    get(
+      "https://content-repo-api-qa.prk-k8s.prd-p6t.org/api/v2/pages/",
+      query: [
+        ["slug", "mnch_onboarding_guided_tour_menu"]
+      ],
+      headers: [["Authorization", "Token @config.items.contentrepo_token"]]
+    )
+
+  page_id = search.body.results[0].id
+
+  content_data =
+    get(
+      "https://content-repo-api-qa.prk-k8s.prd-p6t.org/api/v2/pages/@page_id/",
+      query: [
+        ["whatsapp", "true"]
+      ],
+      headers: [["Authorization", "Token @config.items.contentrepo_token"]]
+    )
+
+  message = content_data.body.body.text.value
+  button_labels = map(message.buttons, & &1.value.title)
+  log("@button_labels")
+end
+
+card DisplayGuidedTourMenu do
+  buttons(
+    CreatProfile: "@button_labels[0]",
+    SpeakToAgent: "@button_labels[1]"
+  ) do
+    text("@message.message")
+  end
+end
+
+card CreatProfile do
+  text("Go to create profile")
+  run_stack("d5f5cfef-1961-4459-a9fe-205a1cabfdfb")
+end
+
+card SpeakToAgent do
+  text("Speak to an agent")
+  run_stack("ea366b74-df7b-41ed-a479-7d501435d38e")
+end
+
+```
+
+# Reminder
+
+```stack
+card Reminder do
+  search =
+    get(
+      "https://content-repo-api-qa.prk-k8s.prd-p6t.org/api/v2/pages/",
+      query: [
+        ["slug", "mnch_onboarding_reminder"]
+      ],
+      headers: [["Authorization", "Token @config.items.contentrepo_token"]]
+    )
+
+  page_id = search.body.results[0].id
+
+  page =
+    get(
+      "https://content-repo-api-qa.prk-k8s.prd-p6t.org/api/v2/pages/@page_id/",
+      query: [
+        ["whatsapp", "true"]
+      ],
+      headers: [["Authorization", "Token @config.items.contentrepo_token"]]
+    )
+
+  message = page.body.body.text.value
+  button_labels = map(message.buttons, & &1.value.title)
+
+  buttons(
+    RemindTomorrow: "@button_labels[1]",
+    RemindNo: "@button_labels[2]"
+  ) do
+    text("@message.message")
+  end
+end
+
+card RemindNo do
+  search =
+    get(
+      "https://content-repo-api-qa.prk-k8s.prd-p6t.org/api/v2/pages/",
+      query: [
+        ["slug", "mnch_onboarding_reminder_later"]
+      ],
+      headers: [["Authorization", "Token @config.items.contentrepo_token"]]
+    )
+
+  page_id = search.body.results[0].id
+
+  page =
+    get(
+      "https://content-repo-api-qa.prk-k8s.prd-p6t.org/api/v2/pages/@page_id/",
+      query: [
+        ["whatsapp", "true"]
+      ],
+      headers: [["Authorization", "Token @config.items.contentrepo_token"]]
+    )
+
+  message = page.body.body.text.value
+  text("@message.message")
+end
+
+card RemindTomorrow do
```

Comment:
This behavior doesn't seem to match what's in the flows. If they click "Later" then we show them a message with a button to Popular Topics.